frontend/src/pages/LandingPage/KeyFeaturesSection.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import drag from '../../assets/LandingPage/KeyFeatures/dragImg.png';
+import flag from '../../assets/LandingPage/KeyFeatures/flag.png';
+import padlock from '../../assets/LandingPage/KeyFeatures/openPadlock.png';
```

Comment:
thoughts on using absolute imports here (`'assets/...'`) so you don't have to find your way back to the root?

Reply:
Oh very true